src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -56,16 +56,16 @@ class AutoImportFix(element: RsElement, private val type: Type) : LocalQuickFixO
     fun invoke(project: Project) {
         val element = startElement as? RsElement ?: return
         val (_, candidates) = when (type) {
-            GENERAL_PATH -> findApplicableContext(project, element as RsPath)
-            ASSOC_ITEM_PATH -> findApplicableContextForAssocItemPath(project, element as RsPath)
+            GENERAL_PATH -> findApplicableContext(project, getPath(element))
```

Comment:
These changes are not needed after using RsImportHelper?

Reply:
Right, thanks.